components/brave_wallet_ui/common/async/lib.ts

```diff
@@ -546,6 +547,8 @@ export function refreshNetworkInfo () {
 export function refreshKeyringInfo () {
   return async (dispatch: Dispatch, getState: () => State) => {
     const { wallet: { selectedCoin } } = getState()
+    const { wallet: { selectedNetwork } } = getState()
```

Comment:
Could be combined into one line
`const { wallet: { selectedCoin, selectedNetwork } } = getState()`